Packs/DigitalShadows/IncidentTypes/incidenttype-Digital_Shadows_Intelligence_Incident.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
@ds-jdcockrill Why do we need 2 types if the same default playbook is associated?
Also, In case you keep only one of the types, you can also remove the classifier (not the mapper) since the type can be configured through the integration instance config.

Reply:
The two types have different fields associated with them - they also mean different things to a user.

- Intelligence incidents are general intelligence information about some threat actor or event
- Client incidents are specific to a particular customer, for instance one of their websites is being impersonated for Phishing purposes, or similar.

They can also contain a slightly different structure (mostly the Impact and Mitigation fields) and so I separated them.

Reply:
Okay, I will leave this decision to you, if you feel it is needed. Please see my latest comment about the layouts - I still believe it can be done with a single layout but you can choose whether to keep the 2 or not.

Reply:
Thanks - I'll keep these the same and think about the layout.